research-software-directory/RSD-as-a-service
# feat: additional package manager logos

# Additional package managers

Closes #1081

Changes proposed in this pull request:
* Additional logos are added to package manager section of software pages
* The status of scraper service is shown for each package manager. The package managers that do not have scrapers show message "RSD scraper services not available".  

How to test:
* `make start` to build app and generate test data
* login as rsd-admin in order to be able to edit existing software items or create new
* navigate to edit software, package managers section and add the following items:
  * Debian: https://packages.debian.org/search?keywords=libcdk-java
  * Sonatype: https://central.sonatype.com/artifact/org.openscience.cdk/cdk-bundle/overview
  * Chocolatey: https://community.chocolatey.org/packages/openchrom
  * Snapcraft: https://snapcraft.io/openchrom
  * Github: https://github.com/orgs/research-software-directory/packages?repo_name=RSD-as-a-service
  * Gitlab: https://gitlab.com/gitlab-org/gitlab/container_registry
  * Rust: https://crates.io/crates/serde
  * Go: https://pkg.go.dev/strconv
* For each of these items the status of the items should mention "RSD scraper services not available" (see image 1)
* Check how the items are presented on the software page (see image 2) 

## Example new package manager icons
![image](https://github.com/research-software-directory/RSD-as-a-service/assets/9204081/f10e07c7-11ba-4dce-9eb3-e2eb9e9e2a5d)

## Example new package manager icons on software page
![image](https://github.com/research-software-directory/RSD-as-a-service/assets/9204081/2567cc7c-2370-413c-8a67-35ba22f95ef3)

PR Checklist:
